### PR TITLE
Add vertex user space and angler2 keyboard

### PR DESCRIPTION
**Pull request by Vertex-kb**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Vertex-kb** [comment]
The modification has been completed. Please approve.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by Vertex-kb** [comment]
@fauxpark Hi,bro!Excuse me, maybe it can be merged? Maybe there is something wrong? Thank you very much for taking your precious time

File: keyboards/vertex/angler2/info.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
+    "keyboard_name": "angler2",
+    "url": "https://github.com/Vertex-kb",
+    "maintainer": "vertex",
+    "layouts": {
+        "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/vertex/angler2/info.json:6]:
```suggestion
        "LAYOUT_all": {
```

File: keyboards/vertex/angler2/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/vertex/angler2/keymaps/default/keymap.c:21]:
```suggestion
  [0] = LAYOUT_all(
```

File: keyboards/vertex/angler2/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    KC_ESC,   KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7 , KC_F8, KC_F9,  KC_F10,  KC_F11, KC_F12, KC_DEL,  KC_PSCR, KC_SLCK,KC_PAUS,
+		KC_GRV,   KC_1,   KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,  KC_NO,  KC_MINS,  KC_EQL, KC_BSPC,   KC_INS, KC_HOME, KC_PGUP,
+		KC_TAB,   KC_Q,   KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,  KC_LBRC, KC_RBRC, KC_BSLS,   KC_DEL, KC_END, KC_PGDN,
+		KC_CAPS,  KC_A,   KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L, KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT,
+		KC_LSFT, KC_SLSH, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,          KC_LSFT,             KC_UP,
+		KC_LCTL, KC_LGUI, KC_LALT,                    KC_SPC,                     KC_LALT, MO(1),   KC_APP, KC_RCTL,    KC_LEFT, KC_DOWN,KC_RGHT
+  ),
+  [1] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/vertex/angler2/keymaps/default/keymap.c:30]:
```suggestion
  [1] = LAYOUT_all(
```

File: keyboards/vertex/angler2/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/vertex/angler2/keymaps/via/keymap.c:21]:
```suggestion
  [0] = LAYOUT_all(
```

File: keyboards/vertex/angler2/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+    KC_ESC,   KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7 , KC_F8, KC_F9,  KC_F10,  KC_F11, KC_F12,   KC_DEL,  KC_PSCR, KC_SLCK,KC_PAUS,
+    KC_GRV,   KC_1,   KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,  KC_NO,  KC_MINS,  KC_EQL, KC_BSPC,   KC_INS, KC_HOME, KC_PGUP,
+    KC_TAB,   KC_Q,   KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,  KC_LBRC, KC_RBRC, KC_BSLS,   KC_DEL, KC_END, KC_PGDN,
+    KC_CAPS,  KC_A,   KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L, KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT,
+    KC_LSFT, KC_SLSH, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,          KC_LSFT,             KC_UP,
+    KC_LCTL, KC_LGUI, KC_LALT,                    KC_SPC,                     KC_LALT, MO(1),   KC_APP, KC_RCTL,    KC_LEFT, KC_DOWN,KC_RGHT
+  ),
+  [1] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/vertex/angler2/keymaps/via/keymap.c:30]:
```suggestion
  [1] = LAYOUT_all(
```

File: keyboards/vertex/angler2/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+    RESET,  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,   _______,_______,_______,
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,   _______,_______,_______,
+    _______,RGB_TOG,RGB_VAI,RGB_VAD,RGB_HUI,RGB_HUD,RGB_MOD,_______,_______,_______,_______,_______,_______,_______,   _______,_______,_______,
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,           _______,
+    _______,_______,_______,                        _______,                        _______,_______,_______,_______,   _______,_______,_______
+  ),
+  [2] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/vertex/angler2/keymaps/via/keymap.c:39]:
```suggestion
  [2] = LAYOUT_all(
```

File: keyboards/vertex/angler2/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+    RESET,  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,   _______,_______,_______,
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,   _______,_______,_______,
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,   _______,_______,_______,
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,           _______,
+    _______,_______,_______,                        _______,                        _______,_______,_______,_______,   _______,_______,_______
+  ),
+  [3] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/vertex/angler2/keymaps/via/keymap.c:48]:
```suggestion
  [3] = LAYOUT_all(
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**